### PR TITLE
Add lighter-video-files rule

### PR DESCRIPTION
**Pull request by martinabab**
#### Pull Request purpose

[ ] Documentation update
[ ] Bug fix
[X] New rule
[ ] Updates an existing rule
[ ] Add something to the core
[ ] Other, please explain:

#### Checklist

[ ] Update CHANGELOG.md
[ ] Update task status in release plan (README)
[ ] Docs added (/docs/rules/<rule_name>)
[ ] Docs ready to be added to website (ec0lint.com)

### CODE REVIEW

File: lib/rules/lighter-video-files.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        type: "problem",
+
+        docs: {
+            description: "Avoid heavy formats of video files",
+            category: "Code improvements",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/lighter-video-files"
+        },
+        messages: {
+          rejected: 'Format of video file can be changed to WebM. CO2 reduction: even 60% of the video file.\n' +
```
Comment (malwinq) [inline lib/rules/lighter-video-files.js]: pls remove CO2 reduction part - there's 'estimated CO2 reduction' later on, so there's no need to duplicate (it was changed recently for other rules too)

**Review by malwinq — COMMENTED** [review]
in general looks good to me, just the log message change

File: lib/rules/lighter-video-files.js (hunk 1 above)
Comment (malwinq) [reply lib/rules/lighter-video-files.js]: you can check lighter-image-files for reference

**Review by malwinq — APPROVED** [review]

**Review by nemmip — APPROVED** [review]
lgtm
Great job!

**Pull request merged**